libraries/botframework-connector/src/auth/skillValidation.ts

```diff
@@ -123,7 +125,7 @@ export namespace SkillValidation {
         channelId: string,
         authConfig: AuthenticationConfiguration): Promise<ClaimsIdentity> {
         if (!authConfig) {
-            throw new Error('SkillValidation.authenticateChannelToken(): invalid authConfig parameter');
+            throw new AuthenticationError('SkillValidation.authenticateChannelToken(): invalid authConfig parameter', StatusCodes.BAD_REQUEST);
```

Comment:
This looks incorrect. The `authConfig` is part of the bot's configuration, it's not received over the wire. Throwing an `INTERNAL_SERVER_ERROR` would be correct here.